### PR TITLE
Add Zonic to `nft.trades` on zkSync Era

### PR DESCRIPTION
**Pull request by tomfutago**

### CODE REVIEW

**Comment by 0xRobin** [comment]
@tomfutago since this is the 3rd use of the Zonic model (Optimism, Scroll, zkSync), I'd like to put it in a macro so we can re-use the logic. Are you up to try and do that here in this PR? 
Otherwise I'll try and get to it next week and we can then incorporate it here.

**Comment by tomfutago** [comment]
> @tomfutago since this is the 3rd use of the Zonic model (Optimism, Scroll, zkSync), I'd like to put it in a macro so we can re-use the logic. Are you up to try and do that here in this PR? Otherwise I'll try and get to it next week and we can then incorporate it here.

Sure, will convert into macro for all 3

**Comment by 0xRobin** [comment]
@tomfutago this already looks good, let me know when you want a final review.

**Comment by tomfutago** [comment]
> @tomfutago this already looks good, let me know when you want a final review.

hey @0xRobin - it's ready for final review now

**Review by 0xRobin — APPROVED** [review]
great work! :ship:

File: models/transfers/zksync/transfers_zksync_eth.sql
Comment (jeff-dude) [inline models/transfers/zksync/transfers_zksync_eth.sql:1]: we don't have to solve in this PR, but we will likely need to deprecate all of these `transfers_<blockchain>_eth` spells in favor of the new `tokens_<blockchain>_transfers` spells

looks like optimism/scroll/zksync are a bit nested in lineages for nft, as we can see in this PR

**Pull request merged**